xarray/backends/zarr.py

```diff
@@ -1768,6 +1768,10 @@ def _get_open_params(
     else:
         missing_exc = zarr.errors.GroupNotFoundError
 
+    if _zarr_v3():
+        if not store.supports_consolidated_metadata:
```

Comment:
```suggestion
        if not getattr(store, "supports_consolidated_metadata", True):
```

I think this will be needed for Zarr >3.0.0,<3.1.0

Reply:
Right - I had something like that then removed it 🤦